bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -44,6 +47,11 @@
 				<description>Command timeout seconds.</description>
 				<default>5</default>
 			</parameter>
+			<parameter name="recordSeconds" type="integer" required="true" min="3" max="20">
```

Comment:
recordTime looks more appropriate to me. 
And in the description: how much time in seconds the... 
If you agree, it has to be changed at different places including documentation.